Gemfile.lock

```diff
@@ -1,75 +1,75 @@
-    actionmailbox (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      activejob (= 6.0.2.2)
-      activerecord (= 6.0.2.2)
-      activestorage (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
+    actionmailbox (6.0.3.2)
+      actionpack (= 6.0.3.2)
+      activejob (= 6.0.3.2)
+      activerecord (= 6.0.3.2)
+      activestorage (= 6.0.3.2)
+      activesupport (= 6.0.3.2)
       mail (>= 2.7.1)
-    actionmailer (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      actionview (= 6.0.2.2)
-      activejob (= 6.0.2.2)
+    actionmailer (6.0.3.2)
+      actionpack (= 6.0.3.2)
+      actionview (= 6.0.3.2)
+      activejob (= 6.0.3.2)
       mail (~> 2.5, >= 2.5.4)
       rails-dom-testing (~> 2.0)
-    actionpack (6.0.2.2)
-      actionview (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
+    actionpack (6.0.3.2)
+      actionview (= 6.0.3.2)
+      activesupport (= 6.0.3.2)
       rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actiontext (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      activerecord (= 6.0.2.2)
-      activestorage (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
+    actiontext (6.0.3.2)
+      actionpack (= 6.0.3.2)
+      activerecord (= 6.0.3.2)
+      activestorage (= 6.0.3.2)
+      activesupport (= 6.0.3.2)
       nokogiri (>= 1.8.5)
-    actionview (6.0.2.2)
-      activesupport (= 6.0.2.2)
+    actionview (6.0.3.2)
+      activesupport (= 6.0.3.2)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activejob (6.0.2.2)
-      activesupport (= 6.0.2.2)
+    activejob (6.0.3.2)
+      activesupport (= 6.0.3.2)
       globalid (>= 0.3.6)
-    activemodel (6.0.2.2)
-      activesupport (= 6.0.2.2)
-    activerecord (6.0.2.2)
-      activemodel (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
-    activestorage (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      activejob (= 6.0.2.2)
-      activerecord (= 6.0.2.2)
+    activemodel (6.0.3.2)
+      activesupport (= 6.0.3.2)
+    activerecord (6.0.3.2)
+      activemodel (= 6.0.3.2)
+      activesupport (= 6.0.3.2)
+    activestorage (6.0.3.2)
+      actionpack (= 6.0.3.2)
+      activejob (= 6.0.3.2)
+      activerecord (= 6.0.3.2)
       marcel (~> 0.3.1)
-    activesupport (6.0.2.2)
+    activesupport (6.0.3.2)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-      zeitwerk (~> 2.2)
+      zeitwerk (~> 2.2, >= 2.2.2)
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
-    autoprefixer-rails (9.7.5)
+    autoprefixer-rails (9.8.6.1)
       execjs
     bindex (0.8.1)
-    bootsnap (1.4.6)
+    bootsnap (1.4.7)
       msgpack (~> 1.0)
     bootstrap (4.4.1)
       autoprefixer-rails (>= 9.1.0)
       popper_js (>= 1.14.3, < 2)
       sassc-rails (>= 2.0.0)
     builder (3.2.4)
-    byebug (11.1.1)
-    capybara (3.31.0)
+    byebug (11.1.3)
```

Comment:
### [byebug](https://github.com/deivid-rodriguez/byebug)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 11.1.1 → 11.1.3

[_~~change-log~~, [source-code](https://github.com/deivid-rodriguez/byebug)_]





<details>
<summary>Commits</summary>

A change of **58** commits. See the full changes on [the compare page](https://github.com/deivid-rodriguez/byebug/compare/v11.1.1...v11.1.3).

These are the first 10 commits:
- (161c098) [Rename build workflow to ubuntu](https://github.com/deivid-rodriguez/byebug/commit/161c098bf2d5f33e5fd6e03595429a7e780501ad)
- (048d83f) [Rename fedora's workflow main job](https://github.com/deivid-rodriguez/byebug/commit/048d83ff00b514a95622f77257cce8d19770c98f)
- (1335bde) [Rename daily workflow and main job](https://github.com/deivid-rodriguez/byebug/commit/1335bde2326911d81f91a80121b2f61a3080952c)
- (3519235) [Merge pull request #639 from deivid-rodriguez/ci/renames](https://github.com/deivid-rodriguez/byebug/commit/3519235a29fe13f9d5abb71f80e45954a34d41d6)
- (5219777) [Remove unnecessary quoting (#640)](https://github.com/deivid-rodriguez/byebug/commit/52197774bc6da445c4b45df41b65648dbfb2b459)
- (87d9a7e) [Use latest byebug image in daily workflow (#641)](https://github.com/deivid-rodriguez/byebug/commit/87d9a7e382ba93aafb9ccbabc7e9600e00319b3f)
- (ab9bf88) [Simplify shelling out helpers in remote debugging tests](https://github.com/deivid-rodriguez/byebug/commit/ab9bf882e92784045817a05685d62884d21d7431)
- (c08feba) [Avoid passing around the error thread too](https://github.com/deivid-rodriguez/byebug/commit/c08feba9a4ac76d3191a0999bda5a0b50fc5b99b)
- (4e359bd) [Only print errors if command fails](https://github.com/deivid-rodriguez/byebug/commit/4e359bd5f7eecc02ad4193d8455bb9582d2babf6)
- (11d28b9) [Print the whole output of command on errors](https://github.com/deivid-rodriguez/byebug/commit/11d28b9dfcc621cd9227f6a3532b4083aa0cc693)


</details>